src/components/calcite-scrim/calcite-scrim.scss

```diff
@@ -1,7 +1,5 @@
 :host {
-  display: flex;
-  position: relative;
-  pointer-events: none;
+  @apply flex relative pointer-events-none;
   --calcite-scrim-background: #{rgba($blk-000, 0.75)};
```

Comment:
It would be ideal if we could kill the rgba() call.

Reply:
What would be the best way to do this?